torchdrivesim/mesh.py

```diff
@@ -235,6 +236,28 @@ def unpickle(cls, mesh_file_path: str, pickle_module: Any = default_pickle_modul
         else:
             raise BadMeshFormat
 
+    def save(self, file_save_path: str):
+        """
+        Save the attributes of the object in a non-pickle format.
+        """
+        if not os.path.exists(os.path.dirname(file_save_path)):
```

Comment:
Does this still work if `dirname` is empty? If not, it's worth adding a check for that. Or maybe there already is a more robust implementation of this function available somewhere in the standard library.